vllm/engine/llm_engine.py

```diff
@@ -1390,6 +1390,12 @@ def _get_stats(
                 for scheduler in self.scheduler)
             cpu_cache_usage_sys = 1.0 - (num_free_cpu / num_total_cpu)
 
+        # Prefix Cache Hit Rate
+        cpu_prefix_cache_hit_rate = self.scheduler[
+            0].block_manager.prefix_cache_hit_rate(Device.CPU)
+        gpu_prefix_cache_hit_rate = self.scheduler[
+            0].block_manager.prefix_cache_hit_rate(Device.GPU)
```

Comment:
nit: should this go into the scheduler so that the coupling of llm engine <--> scheduler is reduced?

do we care about PP here? how to manage that

Reply:
- Moved to scheduler.
- This should also cover PP because the kv-cache in all PP stages are identical (correct me if wrong).